### PR TITLE
Issue #10366: Update inputs for WriteTagCheckTest

### LINKED ISSUE #10366: Update inputs for WriteTagCheckTest
Child of #10080

### PR DESCRIPTION
**Pull request by shashwatj07**
Resolves #10366

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java [hunk 1]
```diff
@@ -178,28 +178,27 @@ public void testRegularExError() throws Exception {
...
 
     @Test
     public void testEnumsAndAnnotations() throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(WriteTagCheck.class);
-        checkConfig.addProperty("tag", "@incomplete");
-        checkConfig.addProperty("tagFormat", ".*");
-        checkConfig.addProperty("severity", "ignore");
-        checkConfig.addProperty("tagSeverity", "error");
-        checkConfig.addProperty("tokens",
+        checkConfig.addAttribute("tag", "@incomplete");
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java:189]: Hmm, can we stick to new name  of method in configuration ?
How can make sure we will not use deprecated method?

**Review by romani — CHANGES_REQUESTED** [review]
Items:

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java (hunk 1 above)
Comment (shashwatj07) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java:189]: > Hmm, can we stick to new name of method in configuration ?

Yes, I'm so sorry. I didn't notice that PR had been merged.

Comment (shashwatj07) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java]: > How can make sure we will not use deprecated method?

We can impose check for it in checkstyle_checks

**Review by romani — APPROVED** [review]

**Pull request merged**